### ISSUE TITLE
GTK theme reload

### ISSUE THREAD

**Issue by koopa1338** [deviantfero/wpgtk#248]
Hi, most applications pick up the theme change directly, but firefox and chromium do not reload the theme on the fly. I looked into closed issue and found that the package `xsettingsd` should be installed if there is no settings daemon. I'm using bspwm, installed `xsettingsd` and created a config file with this content

```conf
Net/ThemeName "FlatColor"
```
if I start `xsettingsd` in my terminal and change my theme in wpgtk I see the following output:

```
xsettingsd: Loaded 1 setting from /home/koopa/.config/xsettingsd/xsettingsd.conf
xsettingsd: Created window 0x1400001 on screen 0 with timestamp 15657407
xsettingsd: Selection _XSETTINGS_S0 is owned by 0x4000001
xsettingsd: Took ownership of selection _XSETTINGS_S0
xsettingsd: 0x4000001 took a selection from us; exiting
xsettingsd: 0x1400001 took a selection from us; exiting
```

Am I doing something wrong? Do I have to start `xsettingsd` as a daemon in the background on startup?

**Comment by nekwebdev** [comment]
I can confirm the same behavior as @koopa1338 but I am using lxappearance-gtk3 and it does reload fine when the wallpaper is changed via the wpg gui.
I do encounter the same issues with wpg gui not reloadind when it is changing a wallpaper, while it does reload in the other cases you stated.

Hope it helps

**Comment by deviantfero** [comment]
I can't speak for the rest of applications in your list @koopa1338 but for me firefox is indeed reloading the theme each time I change theme, are you using a GTK based firefox? I'm using this theme
![1640551314](https://user-images.githubusercontent.com/11150153/147419580-54f1174d-9a2c-43bc-ae2a-ac2520bf6b89.png)

**Comment by koopa1338** [comment]
@deviantfero yes I'm using the GTK version of firefox (exact version is `Mozilla Firefox 96.0.2 20220120170021 20220120170021`). I also checked the themes settings and switched from dark to system them - auto like in your screenshot but that didn't work either.

I experienced some weird issue in firefox though: the theme of the context menus and menus in firefox did change as you can see in this [screenshots](https://imgur.com/a/44ISHvK).

**Comment by deviantfero** [comment]
sorry I haven't replied in a while, was GTK was able to reload on your machine?

**Comment by koopa1338** [comment]
Hey @deviantfero thanks for the reply. No worries about delays ;-) Still facing the same Issue at least regarding Firefox. Meanwhile, I switched to Gentoo and also compiled Firefox from source, but still the same behavior. I think I'm fine with restarting for now, as it is only once or twice a week. I don't know how I could narrow down the root cause at this point. 

Would still love to support regarding this bug/issue, so if you can think of anything that I can test or prepare, let me know.

**Comment by secretrix** [comment]
Can you try to remove ~/.xsettingsd and change the theme? Make sure xsettingsd is *not* running but it is available on the system. Also, use lxappearance to set the theme to linea-nord-color. (if you don't have this theme installed use `wpg-install.sh -G` to install it)
